src/block/image/deprecated.js

```diff
@@ -3,6 +3,25 @@ import { attributes } from './schema'
 
 import { withVersion } from '~stackable/higher-order'
 
+import { addFilter } from '@wordpress/hooks'
+import { semverCompare } from '~stackable/util'
+
+addFilter( 'stackable.image.save.wrapper', 'stackable/newWrapper', ( output, imageWrapperClasses, version, notImageBlock, Wrapper, blockName ) => {
```

Comment:
It's odd that you're padding `notImageBlock` here. `notImageBlock` should be removed from the block component